src/main/java/ua/tc/marketplace/exception/attribute/AttributeNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package ua.tc.marketplace.exception.attribute;
+
+import org.springframework.http.HttpStatus;
+import ua.tc.marketplace.exception.model.CustomRuntimeException;
+
+/** Exception thrown when a specified attribute is not found. */
+public class AttributeNotFoundException extends CustomRuntimeException {
```

Comment:
Please create ItemNoteFoundException (or ModelNotFoundException) and extend all ...NotFoundException from that one. That will simplify exception handling. Because you will need to handle only parent ItemNoteFoundException and not all types

Reply:
For now we have similar mechanism with `CustomRuntimeException`. All our exceptions are extended from it, and `GlobalExceptionHandler` handles only it. So maybe it is enough?